core/integration/steps/the_orders_should_have_the_following_states.go

```diff
@@ -67,6 +81,9 @@ func parseOrdersStatesTable(table *godog.Table) []RowWrapper {
 		"side",
 		"volume",
 		"price",
+		"remaining",
```

Comment:
Why would `remaining` be a required field? It's far easier to add it as an optional one. Tests where we submit an order and check for resulting trades == 0 will automatically have a remaining == size, but this doesn't make a whole lot of sense to include in FOK/IOC or any order that has been filled. Checking the final status is more than enough.

Reply:
Being very explicit in the order state verification brings more confidence in the tests. As for verifying the trading core feature and state, I don't think we are _saving_ anything from not being extremely assertive. That might look redundant but these tests are also here to make us as confident as possible.